modules/message.sh

```diff
@@ -22,6 +22,16 @@ function message {
   }
   function get_channel_slack {
     channel_list=$(curl -s -F token=$access_token -F types=public_channel,private_channel https://slack.com/api/users.conversations)
+    ok=$(echo $channel_list | jq '."ok"')
+    error=$(echo $channel_list | jq '."error"')
+
+    if echo $ok | grep -q "false"; then
+      return 1
+    fi
+    if echo $error | grep -q "missing_scope"; then #needs to check if not successful instead of specific error
```

Comment:
I like this change. I think it would be better if you moved the needed conditions from line 320 and so on to here.

Reply:
This section will only run when `channel_names=$(get_channel_slack)` is successful. (`if channel_names=$(get_channel_slack); then` from line 315.)

If I cut and paste the else statement from line 320 onwards to inside `function get_channel_slack`, it won't run due to the JQ errors causing the conditional on line 315 to skip `function get_channel_slack`.

Reply:
That's fine. I will need to test this again since I just recently found out that my tokens weren't refreshing with different scopes.